src/components/Category/AssetCategoriesComponent.jsx

```diff
@@ -98,55 +94,27 @@ export class AssetCategoriesComponent extends React.Component {
               </ModalComponent>
             </div>
           </div>
-          <Table basic selectable>
-            <Table.Header>
-              <Table.Row>
-                <Table.HeaderCell>Category</Table.HeaderCell>
-              </Table.Row>
-            </Table.Header>
-
-            <Table.Body>
-              {
-                this.props.categories.map(category => (
-                  <TableRow
-                    key={category.id}
-                    data={category}
-                    headings={['category_name']}
-                  />
-                ))
-              }
-            </Table.Body>
 
-            <Table.Footer>
-              <Table.Row>
-                {
-                    !this.emptyCategoriesCheck() && (
-                    <Table.HeaderCell colSpan="3" id="pagination-header">
-                      <Segment.Group horizontal id="art-pagination-section">
-                        <Segment>
-                          <Pagination
-                            totalPages={this.handlePageTotal()}
-                            onPageChange={this.handlePaginationChange}
-                            activePage={this.state.activePage}
-                          />
-                        </Segment>
-                        <Segment>
-                          <DropdownComponent
-                            customClass="page-limit"
-                            placeHolder="Show Rows"
-                            options={rowOptions}
-                            onChange={this.handleRowChange}
-                            value={this.state.limit}
-                            upward
-                          />
-                        </Segment>
-                      </Segment.Group>
-                    </Table.HeaderCell>
-                    )
-                  }
-              </Table.Row>
-            </Table.Footer>
-          </Table>
+          {
+            <Cards
+              data={this.props.categories}
+              headings={['category_name']}
+              imageName="category.svg"
+            />
+          }
+
+          {
+            !this.emptyCategoriesCheck() && (
```

Comment:
Delete this and just render the `<Paginator />`. Looking at the component, `ItemsNotFoundComponent` will be rendered if there are no categories and this block will never be reached. Therefore, instead of having:
```jsx
{ !this.emptyCategoriesCheck() && (
  <Paginator />
)}
```
render:
```jsx
<Paginator />
```

PS: I know the component needs more refactoring but for now, don't go into the "refactor rabbit hole"